attributes/default.rb

```diff
@@ -58,6 +58,9 @@
 default['cfncluster']['nvidia']['enabled'] = 'no'
 default['cfncluster']['nvidia']['driver_url'] = 'http://download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run'
 default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux'
```

Comment:
Could we do the same for cuda driver?

Reply:
Tried but that same version is not present in the mirror

Reply:
ok, thank you!